tasks/src/it/resources/logback.xml

```diff
@@ -0,0 +1,18 @@
+<configuration>
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder>
+            <pattern>%d{HH:mm:ss.SSS} %-5level %logger{36} - "%msg"%n</pattern>
+        </encoder>
+    </appender>
+
+
+    <logger name="org.apache.qpid" level="ERROR"/>
```

Comment:
do we use qpid in bucky's tests?

Reply:
Qpid is a dependency in `build.sbt` and the logger is configured in all sub projects.